sources/whatsnew.asciidoc

```diff
@@ -9,9 +9,21 @@
 
 === What's new in 2.0
 
-The main contribution of Bean Validation 2.0 is leveraging the new language features of Java 8 for the purposes of validation.
-
-TODO
+The main contribution of Bean Validation 2.0 is leveraging the new language features and API additions of Java 8 for the purposes of validation.
+The changes include:
+
+* Support for the new date/time data types for `@Past` and `@Future` (see <<builtinconstraints>>)
+fine-grained control over the current time and time zone used for validation (see <<constraintsdefinitionimplementation-validationimplementation-temporal-validators>>)
```

Comment:
Is there a punctuation missing or something between line 15 and 16? It looks weird.